README.md

```diff
@@ -39,18 +39,28 @@ Set up project with Docker
     docker exec -it linkedevents-backend python manage.py event_import helmet --events
     # Import events from Espoo
     docker exec -it linkedevents-backend python manage.py event_import espoo --events
+    # Import organizations
+    docker exec -it linkedevents-backend python manage.py import_organizations https://api.hel.fi/paatos/v1/organization/ -s helsinki:ahjo
     ```
-6. Update search indexes:
+6. Add datasources helsinki and helfi:
+    * Use django shell and add 'helsinki' and 'helfi' to DataSource
+    * Add audience `docker exec -it linkedevents-backend python manage.py add_helsinki_audience`
+    * In django shell `KeywordSet.objects.create(id='helfi:topics', data_source_id='helfi’)` and add some keywords to helfi:topics
+    * Use Django-admin and allow system datasource to modify data
+
+7. Update search indexes:
     * `docker exec -it linkedevents-backend python manage.py rebuild_index`
     
-7. (Optionally) install API frontend templates:
+8. (Optionally) install API frontend templates:
     * `docker exec -it linkedevents-backend python manage.py install_templates helevents`
 
-8. Start your Django server:
+9. Start your Django server:
     * `docker exec -it linkedevents-backend python manage.py runserver 0:8000`
 
 Now your project is live at [localhost:8000](http://localhost:8000)
 
+For UI, set authentication keys to local_settings.
```

Comment:
Perhaps we should list here separately the backend steps required to use the Helsinki UI, and make it clear that the backend works also without Helsinki UI.